src/cmd/cli/command/commands.go

```diff
@@ -321,8 +321,13 @@ var RootCmd = &cobra.Command{
 			}
 		}
 
-		composeFilePath, _ := cmd.Flags().GetString("file")
-		loader := compose.Loader{ComposeFilePath: composeFilePath}
+		projectName, err := cmd.Flags().GetString("project-name")
+		composeFilePath, err := cmd.Flags().GetString("file")
+		if err != nil || composeFilePath == "" {
+			composeFilePath = filepath.Join(cwd, "compose.yaml")
```

Comment:
This is tricky. The compose spec prefers `compose.yaml` but also accepts `compose.yml`, `docker-compose.yml` and `docker-compose.yaml`. This logic lies in the compose-go library, but we also do our own iteration because we want(ed) to abort when we find more than 1 (see #509 ) I think if we don't have a `--file` we just pass in empty and let the existing logic figure out the actual file name.